test/LootBoxController.test.js

```diff
@@ -44,11 +44,19 @@ describe('LootBoxController', () => {
 
     await deployments.fixture()
 
-    erc20Mintable = await deployContract(wallet, ERC20Mintable, ['test', 'test'])
-    erc721Mintable = await deployContract(wallet, ERC721Mintable, ['test', 'test', 'hello.com'])
-    erc777Mintable = await deployContract(wallet, ERC777Mintable, ['test', 'test', []])
-    erc1155Mintable = await deployContract(wallet, ERC1155Mintable, ['https://hello.com'])
-    lootBox = await deployContract(wallet, LootBox, [])
+    erc20MintableDeploy = await deployments.get('ERC20Mintable')
+    erc20Mintable = await buidler.ethers.getContractAt('ERC20Mintable', erc20MintableDeploy.address, wallet)
+    
+    erc721MintableDeploy = await deployments.get('ERC721Mintable')
+    erc721Mintable = await buidler.ethers.getContractAt('ERC721Mintable', erc721MintableDeploy.address, wallet)
+
+    erc777MintableDeploy = await deployments.get('ERC777Mintable')
+    erc777Mintable = await buidler.ethers.getContractAt('ERC777Mintable', erc777MintableDeploy.address, wallet)
+  
+    erc1155MintableDeploy = await deployments.get('ERC1155Mintable')
+    erc1155Mintable = await buidler.ethers.getContractAt('ERC1155Mintable', erc1155MintableDeploy.address, wallet)
+
+    lootBox = await deployContract(wallet, LootBox, []) // do we want to deploy here or in deloy.js
```

Comment:
@asselstine do we want to deploy a lootbox here or in deploy.js?

Reply:
Let's delete it.  It doesn't seem to be used anywhere.  Cull all dead code! ☠️